api/HTMLOptionElement.json

```diff
@@ -47,6 +47,53 @@
           "deprecated": false
         }
       },
+      "Option": {
+        "__compat": {
```

Comment:
Can you link to https://developer.mozilla.org/en-US/docs/Web/API/HTMLOptionElement/Option and use the description `"<code>Option()</code> constructor"` for this?